10up/ElasticPress
# Fix: EP returns nothing when an invalid value is passed to the tax_query field parameter

### Description of the Change

This PR fixes the issue where the EP returns no post when the invalid value passes to `field` parameter to tax_query. 


Closes #3025 

### How to test the Change

Use the below snippet. It should return the posts

```
add_action( 'init',function() : void {


	$args = array(
		'ep_integrate' => true,
		'posts_per_page'  => 10,
		'tax_query' => array(
			array(
				'taxonomy' => 'category',
				'field'    => 'id', // invalid field name
				'terms'    => termID, // update the value
			),
		),
	);
	$query = new \WP_Query( $args );
	if ( $query->have_posts() ) : ?>
			<?php while ( $query->have_posts() ) : $query->the_post(); ?>
				<?php the_title(); ?>
			<?php endwhile; ?>
	<?php endif; ?>
	<?php wp_reset_postdata();
}  );

```

### Changelog Entry

> Fixed - Returns no posts when an invalid value is passed to the tax_query field parameter


### Credits

Props @burhandodhy 


### Checklist:
